mkdocs.yml

```diff
@@ -121,9 +121,8 @@ nav:
                 - Aggregators: './developer/api/researcher/aggregators.md'
                 - Datasets: './developer/api/researcher/datasets.md'
                 - Experiment: './developer/api/researcher/experiment.md'
-                - Jobs: './developer/api/researcher/jobs.md'
-                - Experiment: './developer/api/researcher/experiment.md'
```

Comment:
I pushed a proposed misc fix for developer API documentation
- remove duplicate `Experiment` entry
- restore alphabetical order